bccm_core/lib/src/features/graphql/client.dart

```diff
@@ -9,32 +9,41 @@ final bccmGraphQLProvider = Provider<GraphQLClient>((ref) {
 class BccmGraphqlHeaders {
   BccmGraphqlHeaders({
     required this.acceptLanguage,
+    required this.acceptAudioLanguage,
+    required this.acceptSubtitleLanguage,
     required this.application,
     required this.applicationVersion,
     required this.featureFlags,
     required this.extraUsergroups,
     required this.isTablet,
     required this.os,
     required this.osVersion,
+    required this.onlyPreferredLanguagesContent,
   });
 
   final List<String> acceptLanguage;
+  final List<String> acceptAudioLanguage;
+  final List<String> acceptSubtitleLanguage;
   final String application;
   final String applicationVersion;
   final List<String>? featureFlags;
   final List<String> extraUsergroups;
   final bool isTablet;
   final String os;
   final String osVersion;
+  final bool? onlyPreferredLanguagesContent;
 
   Map<String, String> toMap() {
     return {
       'Accept-Language': acceptLanguage.join(','),
+      'X-Accept-Audio-Language': acceptAudioLanguage.join(','),
+      'X-Accept-Subtitle-Language': acceptSubtitleLanguage.join(','),
       'X-Application': application,
       'X-Application-Version': applicationVersion,
       'X-Is-Tablet': isTablet ? 'true' : 'false',
       'X-OS': os,
       'X-OS-Version': osVersion,
+      'X-Only-Preferred-Languages-Content': onlyPreferredLanguagesContent == true ? 'true' : 'false',
```

Comment:
Only true if explicitly set to true. In `bccm-mobile`, this will be `null` and therefore `false`